backend/app.js

```diff
@@ -8,7 +8,8 @@ const stockRoutes = require("./routes/stock");
 const app = express();
 mongoose
   .connect(
-    process.env.MONGO_ATLAS //Replace with mongo database url if running locally
+    'mongodb+srv://josephri:TempPass@cluster0.murwd.mongodb.net/bull_market?retryWrites=true&w=majority'
```

Comment:
Change back to process.env.MONGO_ATLAS. So that not exposed to github environment.